db/controllers.js/controllers.js

```diff
@@ -26,3 +31,14 @@ exports.getWholeArticle = (req, res) => {
       next(err);
     });
 };
+
+exports.getCommentsByArticleId = (req, res) => {
+  const id = req.params.article_id;
+  getCommentsByArticleIds(id)
+    .then((comments) => {
+      res.status(200).send({ comments });
+    })
+    .catch((err) => {
+      res.status(err.status).send({ msg: err.msg });
```

Comment:
It would be cleaner to invoke `next` with the error and handle this in the middleware chain.

https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro